LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Have we actually taken MIT License? Please confirm from @prashanthellina , regarding the use of this.

Reply:
@shivam-deepcompute MIT is totally fine.